fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformServiceImpl.java

```diff
@@ -370,77 +305,141 @@ public Collection<ClientData> retrieveActiveClientMembersOfGroup(final Long grou
+            builder.append("c.mobile_no as mobileNo, ");
+            builder.append("c.is_staff as isStaff, ");
+            builder.append("c.email_address as emailAddress, ");
+            builder.append("c.date_of_birth as dateOfBirth, ");
+            builder.append("c.gender_cv_id as genderId, ");
+            builder.append("cv.code_value as genderValue, ");
+            builder.append("c.client_type_cv_id as clienttypeId, ");
+            builder.append("cvclienttype.code_value as clienttypeValue, ");
+            builder.append("c.client_classification_cv_id as classificationId, ");
+            builder.append("cvclassification.code_value as classificationValue, ");
+            builder.append("c.legal_form_enum as legalFormEnum, ");
 
-            sqlBuilder.append("c.closedon_date as closedOnDate, ");
-            sqlBuilder.append("clu.username as closedByUsername, ");
-            sqlBuilder.append("clu.firstname as closedByFirstname, ");
-            sqlBuilder.append("clu.lastname as closedByLastname, ");
+            builder.append("c.submittedon_date as submittedOnDate, ");
+            builder.append("sbu.username as submittedByUsername, ");
+            builder.append("sbu.firstname as submittedByFirstname, ");
+            builder.append("sbu.lastname as submittedByLastname, ");
 
-            sqlBuilder.append("acu.username as activatedByUsername, ");
-            sqlBuilder.append("acu.firstname as activatedByFirstname, ");
-            sqlBuilder.append("acu.lastname as activatedByLastname, ");
+            builder.append("c.closedon_date as closedOnDate, ");
+            builder.append("clu.username as closedByUsername, ");
+            builder.append("clu.firstname as closedByFirstname, ");
+            builder.append("clu.lastname as closedByLastname, ");
 
-            sqlBuilder.append("cnp.constitution_cv_id as constitutionId, ");
-            sqlBuilder.append("cvConstitution.code_value as constitutionValue, ");
-            sqlBuilder.append("cnp.incorp_no as incorpNo, ");
-            sqlBuilder.append("cnp.incorp_validity_till as incorpValidityTill, ");
-            sqlBuilder.append("cnp.main_business_line_cv_id as mainBusinessLineId, ");
-            sqlBuilder.append("cvMainBusinessLine.code_value as mainBusinessLineValue, ");
-            sqlBuilder.append("cnp.remarks as remarks ");
+            // builder.append("c.submittedon as submittedOnDate, ");
+            builder.append("acu.username as activatedByUsername, ");
+            builder.append("acu.firstname as activatedByFirstname, ");
+            builder.append("acu.lastname as activatedByLastname, ");
 
-            sqlBuilder.append("from m_client c ");
-            sqlBuilder.append("join m_office o on o.id = c.office_id ");
-            sqlBuilder.append("left join m_client_non_person cnp on cnp.client_id = c.id ");
-            sqlBuilder.append("join m_group_client pgc on pgc.client_id = c.id ");
```

Comment:
This has been deleted. I know it wasn't used in the select but this is an inner join which implicitly does filtering on the dataset. Move it back pls. Also as I wrote above, please move back everything to its original place.

Reply:
That line is still there in the code this is part of other method that maps a `ClientData` but in this case for Member of Groups

Reply:
In terms of what the diff shows, this isn't there If this is due to reordering again, please undo it and I can check it properly.